### PR TITLE
Update LimitXYAccelJerk

### PR DESCRIPTION
**Pull request by GregValiant**
Add support for RepRap M566 Jerk command (in mm/min).

# Description

The original script only provided for M205.  The change allows a choice of the RepRap M566 command as well as M205.

This fixes... OR This improves... -->

## Type of change

- [ X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Cura 5.4.0 and 4.13.1

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Operating System:  Win 10 Pro

# Checklist:

- [X ] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [ X] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/CONTRIBUTING.md) 
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have uploaded any files required to test this change

### CODE REVIEW

File: plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py [hunk 1]
```diff
@@ -256,11 +278,11 @@ def execute(self, data):
                     lines.insert(len(lines)-2,m205_jerk_old)
                     data[end_index-1] = "\n".join(lines)
                 except:
-                    pass
+                    all
```
Comment (GregValiant) [inline plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py:281]: Should be "Pass".  (hate having to break in a new computer.)

Comment (rburema) [inline plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py:281]:
```suggestion
                    pass
```

File: plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py [hunk 2]
```diff
@@ -31,15 +32,19 @@ def initialize(self) -> None:
...
         machine_name = str(mycura.getProperty("machine_name", "value"))
-                
+        if str(mycura.getProperty("machine_gcode_flavor", "value")) == "RepRap (RepRap)":
+            self._instance.setProperty("jerk_cmd", "value", "reprap_flavor")
+        else:
+            self._instance.setProperty("jerk_cmd", "value", "marlin_flavor")
+
         # Warn the user if the printer is an Ultimaker-------------------------
-        if "Ultimaker" in machine_name:
+        if "Ultimaker" in machine_name or "Griffon" in str(mycura.getProperty("machine_gcode_flavor", "value")):
```
Comment (rburema) [inline plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py:41]:
```suggestion
        if "Ultimaker" in machine_name or "Griffin" in str(mycura.getProperty("machine_gcode_flavor", "value")):
```

I think we spell it 'Griffin' :-)

**Review by rburema — CHANGES_REQUESTED** [review]
Congrats on the new (still uncooperative?) laptop!

One or two small changes should probably do it.

(As an aside, I see I have quite the backlog of things I still need to review from you -- I hope to get some more time for PR's again after the 5.5 release next month... in any case, I'm also away for a week, and should be back around Wednesday or so to finish this small one.)

**Comment by GregValiant** [comment]
So Microsoft sees my new laptop and decides that all my GitHub stuff should be on their "Cloud".  I'm working from my local drive and things aren't matching up with what I see of my files on GitHub.  I KNOW I went through and changed all the "all" to "pass" and my local copy reflects that and the changes for M566.  What's up with that?   Then, I get an e-mail from Microsoft explaining that I've used too much space on the Cloud and I owe them a bunch of money.

I think I have all that figured out now.  PITA.  Here are the final changes on this one.
FYI as you and Casper have perused these I've been making changes to the other post processors.  They aren't professional, but they should be much better.

**Review by rburema — APPROVED** [review]
Seems to be in-order now.

I don't think I'll cherry-pick this to 5.5 though, as we're in feature freeze, and originally the beta was even supposed to come out today (or, well, a week ago...) I really don't want to overload QA on this one!

**Pull request merged**